packages/generator-single-spa/src/root-config/templates/src/index.ejs

```diff
@@ -57,6 +57,7 @@
   <% } %>
 </head>
 <body>
+  <main></main>
```

Comment:
This is needed because otherwise when mounting an app to `document.body` it removes the elements underneath it (system.import, etc.).